storage_test.go

```diff
@@ -0,0 +1,11 @@
+package tprstorage
+
+import (
+	"testing"
+
+	"github.com/giantswarm/microstorage"
+)
+
+func TestInterface(t *testing.T) {
```

Comment:
I created an issue tracking this: https://github.com/giantswarm/giantswarm/issues/1735